pkg/helm/helm.go

```diff
@@ -11,21 +11,25 @@ import (
 
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/chart"
-	"helm.sh/helm/v3/pkg/cli"
 	"helm.sh/helm/v3/pkg/release"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-var settings *cli.EnvSettings = nil
+var (
+	debugOption = (os.Getenv("HELM_DEBUG") == "true")
```

Comment:
We should probably use a real logger instead of these copies from the Helm package, but OK for now.